tests/examples/test_examples.py

```diff
@@ -69,6 +69,7 @@ def report_free_disk_space(capsys):
     yield
 
     with capsys.disabled():
+        # pylint: disable=print-function
```

Comment:
```suggestion
        # pylint: disable-next=print-function
```

Can we use `disable-next`?

Reply:
Done!